CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Keploy UI Contribution Guideline
+
+We're so glad that you're thinking about contributing to Keploy UI. We welcome your contributions!
+
+Here are some guidelines and instructions for contributing to our projects. Please go through them before making a contribution.
+
+## Code of Conduct
+
+Keploy UI governs its participants according to the Contributor [Code of Conduct](https://lfprojects.org/policies/code-of-conduct/). As a contributor, you agree to uphold this code. 
+
+## FAQ and General Questions
+
+Before asking any questions, please have a look at our [Slack Channel](https://join.slack.com/t/keploy/shared_invite/zt-12rfbvc01-o54cOG0X1G6eVJTuI_orSA). 
+
+If your question concerns the code of a project, then you should create a [new issue] in the relevant project's repository.
+
+You can also join chat channel on slack and then ask your question(s) in the relevant channel of our [Slack workspace](https://join.slack.com/t/keploy/shared_invite/zt-12rfbvc01-o54cOG0X1G6eVJTuI_orSA).
+
+## How to Contribute Code?
+
+Bug fixes, performance improvements, code formatting, ...
+There are a lot ways in which you can contribute code!
+The issues list of a project is a great place to find something that you can help us with.
+
+To increase the chances of your code getting merged, please ensure that:
+* Your pull request:
+
+    * Follow all contribution guidelines
+    * Passes all checks and has no conflicts.
+    * Has a well-written title and message that briefly explains your proposed changes.
+
+## How to Report Bugs, Provide Feedback or Request Features?
+
+We welcome all kinds of bug reports, user feedback and feature requests!
+
+We've created some issue templates to assist you in this. Please use them to create [a new issue](https://help.github.com/en/github/managing-your-work-on-github/creating-an-issue) in the relevant project's repository.
```

Comment:
The URL should point to creating a new issue on this repository. Something like https://github.com/keploy/ui/issues/new/choose

Reply:
Done @nehagup please review